### PR TITLE
10.0 fix base multi company hierarchy lmi

### PR DESCRIPTION
**Pull request by lmignon**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Out of curiosity (and a bit off-topic), Laurent, how do you use company hierarchy in a practical way? I only use it for reporting purposes on read-only mode, but it's too dangerous to be used for manipulating master data.

**Comment by Cedric-Pigeon** [comment]
@pedrobaeza I developed an easy share wizard to share products between companies. This wizard allows to copy data from one company to another  such like suppplier infos. I restricted its access to high level users.

**Comment by Cedric-Pigeon** [comment]
@lasley I integrated a commit to @lmignon PR to fix init hook.

File: base_multi_company/hooks.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 def create_company_assignment_view(cr):
     cr.execute("""
         CREATE OR REPLACE VIEW res_company_assignment
-            AS SELECT id, name
+            AS SELECT id, name, parent_id as parent_id
```
Comment (lasley) [inline base_multi_company/hooks.py]: No need to select as if we're targeting the same name:
```
SELECT id, name, parent_id
```

**Comment by lmignon** [comment]
@Cedric-Pigeon Sorry I've lost your changes :-(
Can you force push your local copy????

**Comment by pedrobaeza** [comment]
@Cedric-Pigeon thanks for the explanations!

**Comment by Cedric-Pigeon** [comment]
@lmignon Done
@lasley I introduced 2 fixes. Otherwise I tested with product_mutli_company on my own branch and It works very well

**Review by lasley — APPROVED** [review]
Thanks @lmignon & @Cedric-Pigeon

**Pull request merged**